File: src/main/java/myLang/block/statement/StatementContext.java
package myLang.block.statement;

import myLang.ContextHandler;
import myLang.response.*;
import myLangParser.MyLangParser;

public class StatementContext implements ContextHandler<MyLangParser.StatementContext> {

    @Override
    public Response handler(MyLangParser.StatementContext ctx) throws MyLangException {
        /*if(ctx.block() != null){
            return new myLang.block.BlockContext().handler(ctx.block());
        }*/
        return defaultHandler(ctx);
    }
}

File: src/main/kotlin/myLangKotlin/block/statement/StatementContext.kt
package myLangKotlin.block.statement

import myLangKotlin.ContextHandler
import myLangKotlin.response.MyLangException
import myLangKotlin.response.Response
import myLangParser.MyLangParser

class StatementContext : ContextHandler<MyLangParser.StatementContext> {

    @Throws(MyLangException::class)
    override fun handler(ctx: MyLangParser.StatementContext): Response<*> {
        return defaultHandler(ctx)
    }
}

